targets/csharp/make.js

```diff
@@ -49,6 +52,11 @@ exports.makeCombinedAPI = function (apis, sourceDir, apiOutputDir) {
     }
     generateSimpleFiles(apis, sourceDir, apiOutputDir);
     generateProject(apis, sourceDir, apiOutputDir, "All", ";ENABLE_PLAYFABADMIN_API;ENABLE_PLAYFABSERVER_API;ENABLE_PLAYFABCLIENT_API");
+
+    const xamarinOutputDir = path.join(rootOutputDir, "XamarinTestRunner");
+    var locals = {copyright: copyright};
```

Comment:
Let's insert this line before line 44...

Reply:
I want to double-check before fulfilling this request.  "Declare when needed" is used consistently throughout the rest of this file.  If I change this declaration to "manually hoist", it will become inconsistent with the convention used throughout the rest of the file.

That, of course, begs the question of whether the solution is "then do it everywhere" instead of "never mind, don't do it".  However, I've found that "declare when needed" is generally favored in JS style guides nowadays.  Manual hoisting really only benefits JS-unfamiliar developers who don't understand hoisting, and only in specific scenarios where it makes a functional difference, while "declare when needed" improves the code's self-documenting qualities.